AWSSDKSwiftCLI/Package.swift

```diff
@@ -7,12 +7,14 @@ let package = Package(
     platforms: [
         .macOS(.v13)
     ],
+    products: [
+        .library(name: "AWSCLIUtils", targets: ["AWSCLIUtils"]),
+    ],
```

Comment:
This product is published so `AWSCLIUtils` can be shared with other packages (like the new `SPRCLI` package.)